igor-web/config/igor.yml

```diff
@@ -10,6 +10,13 @@ jenkins:
       username: 'username'
       password: 'password'
 
+dockerRegistry:
```

Comment:
we shouldn't have to update docker registry accounts in two places, ideally the endpoint from clouddriver should be able to return all details.